scripts/e2e.js

```diff
@@ -13,6 +13,8 @@ getUpdatedPackageNames(packageNames => {
     {
       env: Object.assign({}, process.env, {
         PACKAGES: onlyCorePackages,
+        UPDATE_SCREENSHOTS: process.argv[2] === '-i',
+        UPDATE_ALL_SCREENSHOTS: process.argv[2] === '-u',
```

Comment:
I think this may break down when specifying the components to test. You can either run the e2e tests for all components or individual ones.

```bash
# run on changed components using `lerna updated`
yarn e2e

# run on only Button and Link
yarn test:e2e @tds/core-button @tds/core-link
```

So the `process.argv[2]` would break down in the latter case.

We already have the minimist package for parsing command line arguments though, and it is used in "scripts/utils/parseArgs.js". Check out that and see how we can use that here too.